docs/design/deployment.adoc

```diff
@@ -5,7 +5,35 @@ v0.1, 2020-05-17
 
 NOTE: This document is under development
 
-== Migration Plane from Openstack Neutron
+== Migration Plan from Openstack Neutron
+
```

Comment:
The idea of this three-step migration plan is awesome. Let us explicit call it out.

Suggest to use the following subtitles:
 "=== Step 1: Active Neutron and Passive Alcor with Request Caching"
 "=== Step 2: Proactive Request Processing by Alcor"
 "=== Step 3: Seamless Transition to Active Alcor"

Reply:
OK, that will be more clear.